.env

```diff
@@ -1,4 +1,10 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost.bbc.com:7080
```

Comment:
need to run `git checkout latest .env`, commit the change and push and then do `git update-index --assume-unchanged .env` :)